lib/utils/benchmark/src/utilsBenchmark.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2024 Massimo Fioravanti
```

Comment:
you have overwritten the file that was there previously. can you instead create a new file and keep the orginal as well? The rationale is that one day i will probably move this particular beanchmark into a more relevant folder, since it is not really related to the utils folder, but i have not yet decided where.

Reply:
I removed the other file because it give me conflicts when I build with ninja.

> duplicate symbol '_main' in:
>     /Users/sapa/rlc-infrastructure/rlc-release/lib/utils/benchmark/CMakeFiles/utilsBenchmark.dir/src/dictBenchmark.cpp.o
>     /Users/sapa/rlc-infrastructure/rlc-release/lib/utils/benchmark/CMakeFiles/utilsBenchmark.dir/src/utilsBenchmark.cpp.o